docs/content/tutorials/tutorial-compaction.md

```diff
@@ -35,7 +36,7 @@ It will also be helpful to have finished [Tutorial: Loading a file](../tutorials
 
 ## Load the initial data
 
-For this tutorial, we'll be using the Wikipedia edits sample data, with an ingestion task spec that will create a separate segment for each hour in the input data.
+For this tutorial, we'll be using the Wikipedia edits sample data, with an ingestion task spec that will create 1-3 segments per hour in the input data.
```

Comment:
:+1: for the link to segment sizing optimization above, but it would maybe also be worth mentioning that these settings are artificially constrained to produce extra segments so that the example data can be used for the compaction example. I don't think it's required though, so up to you if you modify this section.

Reply:
I think it makes sense. Will add.

Reply:
Added. Thanks!